packages/client/src/pages/SignIn/index.tsx

```diff
@@ -1,3 +1,40 @@
+import { Link } from 'react-router-dom';
+import { RouterList } from '../../router/routerList';
+import { useAuth } from '../../hooks/usuAuth';
+import type { RequestSignIn } from '../../store/auth/type';
+
+const mockParams: RequestSignIn = {
+  login: 'Xxxxx',
+  password: '123XXXxxx',
+};
+
 export default function SignIn() {
-  return <>SignIn</>;
+  const { logout, signIn, checkIsAuth } = useAuth();
+
+  const handleCheckIsAuth = () => {
+    checkIsAuth();
+  };
+
+  const handleSignIn = () => {
+    signIn(mockParams);
+  };
+
+  const handleLogout = () => {
+    logout();
+  };
+
+  return (
+    <>
+      SignIn
+      <button onClick={handleCheckIsAuth}>Проверить авторизацию</button>
```

Comment:
```suggestion
      <button onClick={checkIsAuth}>Проверить авторизацию</button>
```
Нужно сразу передать, не создавая доп. функцию. У `checkIsAuth` нет аргументов

Reply:
Если потребуестя дополнительная функциональности кнопки придется выносить в новую функцию. Да и эти кнопки тут только для проверки работы, они будут заменены на другие. Пока пусть так будет.